enterprise-suite/tests/smoke_nginx

```diff
@@ -48,3 +48,63 @@ test_no_cache() {
 test_no_cache "${CONSOLE}/cluster"
 test_no_cache "${CONSOLE}/workloads/prometheus-server"
 test_no_cache "${CONSOLE}/service/grafana/dashboard/script/exporter-async.js?service-type=cluster"
+
+#
+# security headers
+#
+
+check_header() {
+  val=$( echo "$1" | awk -F ': ' '$1=="'"$2"'"{print $2}' )
+  if [[ -z $val ]]; then
+    echo "missing $2 header"
+    return 1
+  fi
+  if [[ $val != "$3" ]]; then
+    echo "expecting $2: $3, got \"$val\""
+    return 1
+  fi
+  return 0
+}
+
+securitay() {
```

Comment:
Any particular reason for the name?  Would `security` make more sense?

Reply:
![download](https://user-images.githubusercontent.com/1753336/51420007-b54d1f80-1b43-11e9-98ca-90242b78c578.jpg)

Reply:
![image](https://user-images.githubusercontent.com/1753336/51420183-f134b480-1b44-11e9-8750-91d329947997.png)

Reply:
I agree w/ @marcoderama on this one :). This repo is open source and the face of our product after all. Let's keep it professional.